api/v2/log.go

```diff
@@ -41,22 +42,15 @@ func (api *API) LogError(err error, message string, fields ...interface{}) func(
 
 // LogInfo is a wrapper used by the API to handle simple info logs
 func (api *API) LogInfo(message ...interface{}) {
```

Comment:
we can remove these `LogInfo` and `LogDebug` functions entirely now - just use `api.l.Info` and `api.l.Debug`

Reply:
Done!